src/file.ts

```diff
@@ -2747,7 +2746,8 @@ class File extends ServiceObject<File> {
    *
    * const config = {
    *   action: 'read',
-   *   expires: '03-17-2025'
+   *   expires: '03-17-2025',
+   *   accessibleAt: ''03-13-2025'
```

Comment:
Double `'`.

Recommend adding another example separate from the simple Signed URL example.